pkg/scraper/scraper.go

```diff
@@ -0,0 +1,74 @@
+package scraper
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/avast/retry-go"
+	"github.com/tdewolff/minify/v2"
+	"github.com/tdewolff/minify/v2/html"
+)
+
+const (
+	JobCenter = 1
+	Bruneida  = 2
+)
+
+func getDocument(url string) (*goquery.Document, error) {
+	var doc *goquery.Document
+
+	err := retry.Do(func() error {
+
+		res, err := http.Get(url)
+		if err != nil {
+			return err
+		}
+
+		defer res.Body.Close()
+
+		if res.StatusCode != 200 {
+			errorMessage := fmt.Sprintf("status code error: %d %s", res.StatusCode, res.Status)
+
+			log.Fatal(errorMessage)
```

Comment:
I think you're looking for `log.Error`? `log.Fatal` will kill the program.

Reply:
log.Error is not in the standard log package